store/mongo/migration_2_0_1.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2022 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package mongo
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
+	mstore "github.com/mendersoftware/go-lib-micro/store/v2"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	mopts "go.mongodb.org/mongo-driver/mongo/options"
+)
+
+type migration_2_0_1 struct {
```

Comment:
HEADS UP! This will conflict with the 2.0.1 migraiton in enterprise

Reply:
@alfrunes thanks, renamed to 2.0.2

Reply:
Sorry, brain-fart :brain::dash: ... I was confused as I was working with the deployments service in parallel. Disregard.